python/generate_classification.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+cd ${DIR}
+
+CLASSIFICATION_MODEL=ml4ir/applications/classification
+
+PYTHONPATH=. python ${CLASSIFICATION_MODEL}/pipeline.py \
+    --data_dir ${CLASSIFICATION_MODEL}/tests/data/csv/  \
+    --feature_config ${CLASSIFICATION_MODEL}/tests/data/configs/feature_config.yaml \
+    --model_config ${CLASSIFICATION_MODEL}/tests/data/configs/model_config.yaml \
+    --run_id end_to_end_classif \
```

Comment:
This name needs to be factorized I guess.

Reply:
`e2e`? I don't think this will block us

Reply:
I meant that `end_to_end_classif` will create a directory that is then used in the jvm tests. This should be a param from the build system, passed to this script.